src/test/java/org/apache/maven/shared/artifact/filter/AbstractPatternArtifactFilterTest.java

```diff
@@ -19,20 +19,22 @@
  * under the License.
  */
 
-import static org.junit.Assert.assertFalse;
```

Comment:
Those were changed by the formatter, but this is in line with coding conventions.